rebar.config

```diff
@@ -15,4 +15,4 @@
             warn_export_vars,
             warn_obsolete_guard,
             warn_untyped_record,
-            warn_unused_import]}.
+            warn_unused_import]}.
```

Comment:
Seems like the only things that's changed here is the removal of the trailing line break. It's minor but having a trailing line break is a convention in linux.